examples/sync/src/bin/client.rs

```diff
@@ -0,0 +1,253 @@
+//! This client demonstrates how to use the ADB sync functionality
```

Comment:
nit: In these comments, let's replace `ADB` with `[commonware_storage::adb::Any]`